### DISCUSSION TITLE
How do I spawn an existing prototype from script?

### DISCUSSION THREAD

**Discussion by RodRitter** [evennia/evennia#2663] [Evennia Questions]
Hey all, I'm back with another clueless noob question.

I've got a list of prototypes in `prototypes.py` and I want to know how I can spawn from that list, from a room function that I'm using. I only know how to spawn via command, so this is new to me.

Update: Nevermind, I just read the comments in the prototypes file

**Comment by Griatch** [comment]
You can also read the documentation about prototypes: https://www.evennia.com/docs/latest/Spawner-and-Prototypes.html?highlight=spawn#using-evennia-prototypes-spawner
